common-lib/src/main/java/com/google/cloud/tools/intellij/feedback/PluginStatusBundle.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.feedback;
+
+import com.intellij.CommonBundle;
+
+import org.jetbrains.annotations.NonNls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.PropertyKey;
+
+import java.lang.ref.Reference;
+import java.lang.ref.SoftReference;
+import java.util.ResourceBundle;
+
+/**
+ * Resource bundle for messages related to Google Feedback error reporting.
+ */
+public class PluginStatusBundle {
```

Comment:
The main reason was because the code that uses the messages in this bundle (the `DisablePluginWarningDialog`), belongs (in my opinion) in the common-lib module, since it is common (or, has the potential to be) to both the google-cloud-tools and the google-account plugins.  It followed that the resource bundle that it used should also be in common-lib, and I didn't think it made sense to reuse the already-existing `ErrorReporterBundle`, so I created a new one.

An alternative would be to merge `PluginStatusBundle` and `ErrorReporterBundle`, and call it something like `CommonMessageBundle`. Let me know if you think that would be more appropriate.